### DISCUSSION TITLE
How to Enforce Minimum TLS Version for Webhooks?

### DISCUSSION THREAD

**Discussion by sshiba** [kubernetes-sigs/kubebuilder#3862] [Support]
I have a mandate to enforce minimum TLS version to 1.2 and looking into how I can achieve this with KubeBuilder.
I appreciate your help.

**Comment by sshiba** [comment]
Nevermind, I got the answer from #kubebuilder Slack community.

**Comment by camilamacedo86** [comment]
Can you please post the answer here and close this one?
So that you can help others?

**Comment by camilamacedo86** [comment]
Hi @sshiba 

I think what you are looking in what we scaffold by default in the main.go. Example:

https://github.com/kubernetes-sigs/kubebuilder/blob/2b25adec1ffb8e811a85e8d86357adcf7213bf5b/testdata/project-v3/main.go#L73-L87

I hope that answer your question and help you out.
Due it be open for a while without interaction and the answer above might sort out this one, I am closing it and I hope that you do not mind. However, please feel free to re-open if you see that fits.
